UefiCpuPkg/PiSmmCpuDxeSmm/CpuS3.c

```diff
@@ -233,7 +233,6 @@ InitSmmS3ResumeState (
     }
 
     SmmS3ResumeState->SmmS3Cr0 = (UINT32)AsmReadCr0 ();
-    SmmS3ResumeState->SmmS3Cr3 = Cr3;

```

Comment:
could the input parameter of InitSmmS3ResumeState be also removed ?

Reply:
agree